assets/cart.js

```diff
@@ -100,6 +100,7 @@ class CartItems extends HTMLElement {
       .then((state) => {
```

Comment:
When you first get to the page it shows as `sold out` and gets fixed once you switch variants:

https://screenshot.click/31-33-ef46b-6iy9h.mp4

Reply:
Good catch, the Liquid logic for disabling add-to-cart button was [updated](https://github.com/Shopify/dawn/pull/2255), but our logic for variant switching wasn't. Updated - https://github.com/Shopify/dawn/pull/2251/commits/a3cc6f65f1db654ac2d1206c12a207f44c56295a